### PR TITLE
Metachromium docs

### PR DESCRIPTION
**Pull request by yahengsu**
Added some initial documentation about metachromium. Not sure if my understanding is 100% technically correct, so feedback is highly welcome!

### CODE REVIEW

File: docs/metachromium-overview.md [hunk 1]
```diff
@@ -3,5 +3,54 @@ id: metachromium-overview
...
-Metachromium
+[Metachromium](https://metachromium.com/) is a spatial app browser that runs side-by side with OpenVR. It lets you open any WebXR page as a spatial overlay app that runs on top of any other VR game.
 
+## Features
+
+- Run WebXR context on top of an OpenVR application
+
+- Run multiple WebXR applications inside a single Metachromium context
+  
+- Add multiplayer to singleplayer apps using metaRTC
```
Comment (Zetaphor) [inline docs/metachromium-overview.md:14]: MetaRTC is not a feature of Metachromium, it is distributed as a normal XRPackage that can be added to your scene. MetaRTC needs to be covered in a dedicated page.

File: docs/metachromium-overview.md [hunk 2]
```diff
@@ -3,5 +3,54 @@ id: metachromium-overview
...
+
+4. Enjoy!
+
+## How It Works
+
+Metachromium is based on the open source Chromium browser and is based on web standards like WebXR.
+
+### Rendering Content
+
+For different WebXR contexts to be overlayed inside an OpenVR application, special math must be done so the outlines of these different applications blend into the world correctly in terms of its alpha (opacity). Metachromium does the blending such that the renderings appear as though they are in the correct space when viewed through the headset. Functionally, there are a bunch of 3D geometry calculations that are done under the hood to make an immersive experience possible.
```
Comment (Zetaphor) [inline docs/metachromium-overview.md:42]: I believe we can be more specific if we're going to give a technical explanation. Metachromium is capturing a 360 stereoscopic view from the WebXR frame buffer and then mapping that view to a transparent sphere that surrounds the user. A bit like a dynamic photosphere. [Here is a link](https://youtu.be/YGsXRSEemRw?t=1896) to the timestamp where Avaer covered this in the overview

File: docs/metachromium-overview.md [hunk 3]
```diff
@@ -3,5 +3,54 @@ id: metachromium-overview
...
+
+### Using XRPackages
+
+One of the caveats with overlaying WebXR objects is that all WebXR content appears as though it's overlayed on top of the native app, instead of appearing as though it were native to the application. This is one of the main reasons why using XRPackages is preferred, as a `.wbn` generated by XRPackage contains the correct depth rendering information needed for a more immersive rendering.
+
+### Further Reading
+
+To learn more about Metachromium and how to develop your own content, follow our [create your first aug](creating-an-aug.md) tutorial. Other useful links to learning more about the Webaverse, XRPackage, and XR development can be found below.
```
Comment (Zetaphor) [inline docs/metachromium-overview.md:50]: For the sake of styling consistency, let's wrap all page links that use the actual page title in backticks (`example`) and capitalize case the link text to reflect the title.

**Comment by yahengsu** [comment]
Gotcha, thanks for the great feedback! Will work on making those changes :)

File: docs/metachromium-overview.md (hunk 3 above)
Comment (yahengsu) [reply docs/metachromium-overview.md:50]: Hey @Zetaphor just to confirm, did you mean something like
```markdown
follow our [`Create Your First Aug`](creating-an-aug.md) tutorial.
```
So it shows up like [`Create Your First Aug`](creating-an-aug.md)?

Comment (Zetaphor) [reply docs/metachromium-overview.md]: Correct, see [this section on the main page](https://docs.webaverse.com/docs/index.html#import-existing-creations) for an example in production

**Pull request merged**